pandas/core/categorical.py

```diff
@@ -356,19 +358,10 @@ def __init__(self, values, categories=None, ordered=None, dtype=None,
 
             codes = _get_codes_for_values(values, dtype.categories)
 
-            # TODO: check for old style usage. These warnings should be removes
```

Comment:
remove these independently (IOW another PR), it can be before or after this one.

Reply:
OK